source/jormungandr/jormungandr/interfaces/v1/fields.py

```diff
@@ -702,7 +702,7 @@ def output(self, key, value):
 
 route = deepcopy(generic_type)
 route["links"] = DisruptionLinks()
-route["is_frequence"] = fields.String
+route["is_frequence"] = fields.Boolean()
```

Comment:
This look like a non backward compatible change

Reply:
Test swagger fails without this change.